### ISSUE TITLE
NVDA doesn't read a comma in float numbers in input fields.

### ISSUE THREAD

**Issue by syki1** [nvaccess/nvda#13273]
Screen reader doesn't read a comma when second number (after the comma) is from range 31-39. (e.g. 0,36, 1,34, 2,35, 3,32). 

### Steps to reproduce:
1. In text or search type input field write "1,35". (can use example websites).  
2. Click outside a field by mouse or keyboard to lose focus.
3. Click on the field by mouse or keyboard with text to regain focus. Screen reader will read information about field.
https://www.w3schools.com/html/html_forms.asp - example website with text input
https://www.gouvernement.fr/en/news  - example website with search input 
### Actual behavior:
Screen reader doesn't read a comma. Result is "one thirty five".
### Expected behavior:
Screen reader should read "one comma thirty five".
#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
2020.3
#### Windows version:
Win 10 Enterprise x64
#### Name and version of other software in use when reproducing the issue:
Chrome 97.0.4692.71 (Official Build) (64-bit)
#### Other information about your system:
Win 10 with Touch point
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I haven't checked. Limited access to local administrator.

**Comment by feerrenrut** [comment]
This will depend on the [current symbol level](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#SpeechSettingsSymbolLevel). You can change it with `NVDA+p`.

**Comment by mwhapples** [comment]
I somewhat agree with the original request and do not feel suggesting punctuation level is the answer. My reasoning is that some countries use comma for decimal point (eg. I have seen it for prices on German shop websites). Decimal point speaking in NVDA is not affected by punctuation level (IE. 1.35 is always spoken as 1 point 35). I say somewhat agree with the original request because should there be 3 digits after the comma then should the comma be a separator or a decimal point and how can NVDA work that out. Language of the document is not a guarantee, I have found documents in English from German companies still using the comma for decimal point.

**Comment by Adriani90** [comment]
This is not reproducible anymore with Chrome 142 and NVDA 2025.3.1. Closing as works for me. if you are still having this issue, please comment and we can reopen.
